plugin-server/src/cdp/hog-function-manager.test.ts

```diff
@@ -1,52 +1,263 @@
-import { Hub } from '../types'
-import { HogFunctionManager } from './hog-function-manager'
+import { HogFunctionManager } from '~/src/cdp/hog-function-manager'
+import { HogFunctionType, IntegrationType } from '~/src/cdp/types'
+import { Hub } from '~/src/types'
+import { closeHub, createHub } from '~/src/utils/db/hub'
+import { PostgresUse } from '~/src/utils/db/postgres'
+import { insertHogFunction, insertIntegration } from '~/tests/cdp/fixtures'
+import { createTeam, resetTestDatabase } from '~/tests/helpers/sql'
```

Comment:
question: does it mean we can't use relative imports anymore or is it just preferrence? Just curious.